windows/windows_online_updates_install/install_windows_online_updates.yml

```diff
@@ -44,15 +44,28 @@
     skip_reason: "Not Applicable"
   when:
     - not win_check_is_updated
+    - win_online_updates_succeed
     - not win_online_updates_installed
 
-# Fail test case when no update installed while expect guest OS to be updated
-- name: "No Windows updates installed in guest OS"
```

Comment:
If the expected build number is not set and no need to check the OS build number, I think we can just print the not installed windows updates list, no need to fail the test case.

Reply:
Will not check build number after updates install when `windows_updated_build_num` is not set, but updates install should succeed.